cmd/adapter-rest/startcmd/start.go

```diff
@@ -67,11 +68,21 @@ const (
 	hydraURLFlagUsage = "Base URL to the hydra service." +
 		"Alternatively, this can be set with the following environment variable: " + hydraURLEnvKey
 	hydraURLEnvKey = "ADAPTER_REST_HYDRA_URL"
+
+	modeFlagName  = "mode"
+	modeFlagUsage = "Mode in which the edge-adapter service will run. Possible values: " +
+		"['issuer', 'rp']."
+	modeEnvKey = "ADAPTER_REST_MODE"
 )
 
 // API endpoints.
 const (
-	uiEndpoint = "/ui"
+	uiEndpoint       = "/ui"
```

Comment:
should rename to `rpUIEndpoint`

Reply:
Used same endpoint for both issuer/rp as only one mode is supported at a time.